app/static/js/app.js

```diff
@@ -131,6 +131,10 @@ function isIgnoredKeystroke(keyCode) {
 
 // Send a keystroke message to the backend, and add a key card to the web UI.
 function sendKeystroke(keystroke) {
+  //Ignore if keyCode is 229 (mobile)
+  if (keystroke.keyCode === 229) {
```

Comment:
Can we add more detail here about what keycode 229 is? Also, since we reference it in multiple places, can we define it as a named constant instead?

Reply:
keyCode 229 is some kind of placeholder for a combined character, I think it has to do with the autocomplete in the mobile OS. It indicates something is sent but it's not quite done. I got around this by blurring the input essentially clearing the autocomplete and then re-focusing so we can accept the next character.